src/Orleans.Transactions.Azure/Orleans.Transactions.Azure.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <PackageId>Microsoft.Orleans.Transactions.Azure</PackageId>
+    <Title>Microsoft Orleans Transactions on Azure</Title>
+    <Description>Azure Transaction library of Microsoft Orleans used on the server.</Description>
+    <PackageTags>$(PackageTags) Azure Transactions</PackageTags>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <AssemblyName>Microsoft.Orleans.Transactions.Azure</AssemblyName>
+    <RootNamespace>Orleans.Transactions.Azure</RootNamespace>
+    <OrleansBuildTimeCodeGen>true</OrleansBuildTimeCodeGen>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="WindowsAzure.Storage" Version="$(WindowsAzureStorageVersion)" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Orleans.Core.Abstractions\Orleans.Core.Abstractions.csproj" />
```

Comment:
nit: P2P references are also transitive, no need to specify 4 projects here, Orleans.Transactions would be suffice. If you choose to leave it like this we can clean all of these up later.